pages/linux/!.md

```diff
@@ -11,7 +11,7 @@
 
 `!{{number}}`
 
-- Substitute with a command that was used specified lines back:
+- Substitute with a command that was used specified number of lines back:
```

Comment:
Thanks for your contribution.

How about this?

```suggestion
- Substitute with a command that uses a specified number of lines back:
```

Reply:
I feel like that's not grammatically correct.

Reply:
I'm pretty sure it's grammatically correct, but I'd interpret it to mean something different.

My suggestions would be:

```suggestion
- Substitute with a command that was used a specified number of lines back:
```

```suggestion
- Substitute with a command from a specified number of lines back:
```